packages/insured-bridge-relayer/src/Relayer.ts

```diff
@@ -108,6 +123,14 @@ export class Relayer {
     }
   }
 
+  private async isRelayValid(relay: Relay, deposit: Deposit): Promise<boolean> {
+    return (
+      relay.realizedLpFeePct.toString() ===
+        (await this.l1Client.calculateRealizedLpFeePctForDeposit(deposit)).toString() &&
+      relay.depositContract === deposit.depositContract
```

Comment:
Testing an incorrect `depositContract` is a bit tricky. I'm not sure how to do it because the L1BridgePool links the L2BridgeDepositBox address at the time of the relay. Is this either a param that we don't need to check in this method or is there a way to test it?

Reply:
ye, I think we need to validate other params as well. Do you see a way we could use the `InsuredBridgePriceFeed` for this? Otherwise, we need to validate some of the other params as well IMHO.